### PR TITLE
PhyloGibbs-1.2-goolf-1.4.10.eb (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/c805063f4e69dac6676a for a full test report.

**Comment by fgeorgatos** [comment]
@pescobar: potential glib deps are visible here:
https://github.com/hpcugent/easybuild-easyconfigs/tree/master/easybuild/easyconfigs/g/GLib

File: easybuild/easyconfigs/p/PhyloGibbs/PhyloGibbs-1.2-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/PhyloGibbs/PhyloGibbs-1.2-goolf-1.4.10.eb]: minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/c5baa55e9e0afcdbf163 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 3 out of 7 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/23004241afa551494ded for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 4 out of 7 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/394840995b5486478fa7 for a full test report.

**Comment by rjeschmi** [comment]
glib missing?

**Comment by pescobar** [comment]
error seems related to gettext download url. gonna try to fix it....

```
== 2015-09-23 13:11:31,221 main.filetools WARNING Download of http://ftpmirror.gnu.org/gettext/gettext-0.18.2.tar.gz to /software/experimental/sources/g/gettext/gettext-0.18.2.tar.gz failed, done trying
== 2015-09-23 13:11:31,229 main ERROR Couldn't find file gettext-0.18.2.tar.gz anywhere, and downloading it didn't work either... Paths attempted (in order): /scratch/rob/eb-dev/easybuild-easyconfigs/easybuild/easyconfigs/g/gettext/g/gettext/gettext-0.18.2.tar.gz, /scratch/rob/eb-dev/easybuild-easyconfigs/easybuild/easyconfigs/g/gettext/gettext/gettext-0.18.2.tar.gz, /scratch/rob/eb-dev/easybuild-easyconfigs
```

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 3 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/cb391acbdf23d5ab548a for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 2 out of 3 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/1409ab72f4941fa85233 for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/ad9a726260f9fc7f41cd for a full test report.

**Comment by rjeschmi** [comment]
I don't think the source exists at:

http://www.biozentrum.unibas.ch/~nimwegen/phylogibbs/phylogibbs_source.tar.gz

**Comment by boegel** [comment]
ping on this?

Jenkins: test this please

**Comment by boegel** [comment]
+1, can't seem to find the sources anymore...

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months, sources no longer discoverable.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**